package.json

```diff
@@ -145,6 +145,7 @@
     ]
   },
   "fastbootDependencies": [
+    "abortcontroller-polyfill",
```

Comment:
solves `Package 'abortcontroller-polyfill' is required to be in the whitelist.` warnings